tests/components/enphase_envoy/test_switch.py

```diff
@@ -146,12 +145,24 @@ async def test_switch_grid_operation(
 
 
 @pytest.mark.parametrize(
-    ("mock_envoy"), ["envoy_metered_batt_relay"], indirect=["mock_envoy"]
+    ("mock_envoy", "entity_states"),
+    [
+        (
+            "envoy_metered_batt_relay",
+            {
+                "NC1": (STATE_OFF, 0, 1),
```

Comment:
Since we're testing all the entities in the same test run, this dict could just be a (constant) variable in the test or a constant.

Reply:
That is correct, but in case we decide to add more fixtures for mock_envoy (currently only "envoy_metered_batt_relay") in the future then this structure provides that flexibility I'd say.